paymentsheet/src/main/java/com/stripe/android/paymentelement/confirmation/ConfirmationModule.kt

```diff
@@ -0,0 +1,26 @@
+package com.stripe.android.paymentelement.confirmation
+
+import com.stripe.android.paymentsheet.paymentdatacollection.bacs.BacsMandateConfirmationLauncherFactory
+import com.stripe.android.paymentsheet.paymentdatacollection.bacs.DefaultBacsMandateConfirmationLauncherFactory
+import dagger.Binds
+import dagger.Module
+import dagger.Provides
+
+@Module
+internal interface ConfirmationModule {
+    @Binds
+    fun bindsConfirmationHandlerFactory(
+        defaultConfirmationHandlerFactory: DefaultConfirmationHandler.Factory
+    ): ConfirmationHandler.Factory
+
+    @Binds
+    fun bindsIntentConfirmationInterceptor(
+        defaultConfirmationHandlerFactory: DefaultIntentConfirmationInterceptor
+    ): IntentConfirmationInterceptor
+
+    companion object {
+        @Provides
+        fun providesBacsMandateConfirmationLauncherFactory(): BacsMandateConfirmationLauncherFactory =
+            DefaultBacsMandateConfirmationLauncherFactory
+    }
+}
```

Comment:
I'll eventually rename this to `BaseConfirmationModule` which will import all the base confirmation requirements (just intent handling). This will split out into multiple different importable modules for each Payment Option type in which each product can build a custom confirmation module from.